dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -101,19 +133,27 @@ class TaskGridFilter extends FilterPropertySource {
     return true;
   }
 
-  static Map<String, String?>? _stagePropertiesMap;
-  Map<String, String?> get _stageProperties => _stagePropertiesMap ??= <String, String?>{
-        StageName.luci: _luciProperty.fieldName,
-      };
-
   /// Check the values in the [QualifiedTask] for compatibility with the properties of this
   /// filter and return [true] iff the commit column should be displayed.
   bool matchesTask(QualifiedTask qualifiedTask) {
     if (!_taskProperty.matches(qualifiedTask.task!)) {
       return false;
     }
+    Map<String, String> showOSs = {
+      "showMac": "mac",
```

Comment:
Prefer single quotes.

While at it, can you add https://dart-lang.github.io/linter/lints/prefer_single_quotes.html to the analysis_options.yaml? It seems this got dropped when moving over the recommended dart lints